default.nix

```diff
@@ -65,12 +65,13 @@ let
       config' = config;
       crossSystem' = crossSystem;
     in { args ? {}
+       , overlays ? [ jemallocOverlay ]
        , extraOverlays ? nixpkgsOverlays
        , system ? system'
        , globalConfig ? globalConfig'
        , config ? config'
        , crossSystem ? crossSystem' }: import nixpkgs ({
-          overlays = [ jemallocOverlay ] ++ extraOverlays;
+          overlays = overlays ++ extraOverlays;
```

Comment:
Wouldn't we want `overlays ++ [ jemallocOverlay ] ++ extraOverlays`? lest we loose the `jemallocOverlay` once someone passes in the `overlays` arg?

Reply:
yeah, though existing behavior is preserved. only a new capacity to set all overlays is added.

Reply:
ah also, apparently `jemallocOverlay` is only useful for `18.03`, the new jemalloc version that fix the issue is the default since `18.09`.